packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -426,7 +430,7 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
         return innerActivity;
       },
     );
-    return Future.wait<Null>(resultFutures);
+    await Future.wait<Null>(resultFutures);
```

Comment:
Future.wait doesn't return null, so Flutter will throw an error in the logs complaining about the type mismatch.